README.md

```diff
@@ -1,77 +1,80 @@
 ## Firefox Application Services
 
-_A platform for building cloud-powered applications that target Firefox users._
+Application Services is collection of Rust Components that are built together into the application-services (a-s) library to enable Firefox applications to integrate with Firefox accounts, sync, push and experimentation. Each component is built using a core of shared code written in Rust, wrapped with native language bindings for different platforms.
 
-### What's this all about?
+### Contributing
+To contribute, please review the Mozilla [Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/) and then visit our [how to contribute](docs/contributing.md) guide.
 
-This repository hosts the code and docs needed to integrate with the products offered by the Firefox
-[Application Services](https://mana.mozilla.org/wiki/display/CLOUDSERVICES/Application+Services+Home)
-team.
+### Contact
+Get in touch with other community members on Matrix, the mailing list or through issues here on GitHub.
+- Matrix: [#sync:mozilla.org](https://chat.mozilla.org/#/room/#sync:mozilla.org) ([How to connect](https://wiki.mozilla.org/Matrix#Connect_to_Matrix))
+- Mailing list: application-services@mozilla.com
```

Comment:
Is this list open to the public? I recall many of our `@mozilla.com` aliases being internal-only.

Reply:
According to the current settings - *anyone on the web* can post https://groups.google.com/a/mozilla.com/g/application-services/settings.  With the switch to groups - I think it would make sense to re-think our group settings and the intention of this mailing list - but it seems simplest that we use a name that matches the repository name for simplicity.

Reply:
removed the reference to the google group per our team conversation yesterday.